libs/domain/search/src/services/resolvers/breadcrumbs.resolver.ts

```diff
@@ -0,0 +1,79 @@
+import { INJECTOR, Provider, inject } from '@spryker-oryx/di';
+import { Facet, FacetValue } from '@spryker-oryx/product';
+import { RouteType } from '@spryker-oryx/router';
+import {
+  Breadcrumb,
+  BreadcrumbsResolver,
+  BreadcrumbsResolvers,
+  LinkService,
+} from '@spryker-oryx/site';
+import {
+  Observable,
+  combineLatest,
+  map,
+  of,
+  switchMap,
+  throwError,
+} from 'rxjs';
+import { FacetListService } from '../facet-list.service';
+
+export type FlatFacet = { name: string; value: string };
```

Comment:
it it internal type lt's not export it
```suggestion
type FlatFacet = { name: string; value: string };
```